CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# v2.3.1 (2024-08-01)
+
+## OS Changes
+
+* Update docker-engine to v25.0.6 ([#55])
+
+## Build Changes
+
+* Update bottlerocket-settings-models to v0.2.0 ([#58])
+* Update bottlerocket-sdk to v0.43.0 ([#60])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Support hostname-override-source ([#59])
+
+[#55]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/55
+[#58]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/58
+[#59]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/59
+[#60]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/60
+
```

Comment:
```suggestion
[#48]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/48
```

Reply:
Ack, will commit this on my own just to preserve number ordering (before 58)